eng/targets/TargetFrameworks.props

```diff
@@ -13,10 +13,10 @@
   -->
   <PropertyGroup>
     <NetRoslyn>net8.0</NetRoslyn>
-    <NetRoslynAll>net7.0;net8.0</NetRoslynAll>
+    <NetRoslynAll>net8.0</NetRoslynAll>
```

Comment:
TBD - looks like NetPrevious (sourcebuild) is still .net7, but our build gives errors if we have a net7 TFM (probably from the SDK update).